deployment_scripts/linode-marketplace-influxdb/influxdb-deploy.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+set -e
+trap "cleanup $? $LINENO" EXIT
+
+## InfluxDB Settings
+#<UDF name="soa_email_address" label="Email address (for the Let's Encrypt SSL certificate)" example="user@domain.tld">
+#<UDF name="admin_username" label="Admin Username" example="The username for the InfluxDB admin user" default="" required="true">
+#<UDF name="org_name" label="Organization Name" example="The name of the InfluxDB organization" default="" required="true">
+#<UDF name="bucket_name" label="Bucket Name" example="The name of the InfluxDB bucket" default="" required="true">
+
+## Linode/SSH Security Settings
+#<UDF name="user_name" label="The limited sudo user to be created for the Linode: *No Capital Letters or Special Characters*">
+#<UDF name="disable_root" label="Disable root access over SSH?" oneOf="Yes,No" default="No">
+
+## Domain Settings
+#<UDF name="token_password" label="Your Linode API token. This is needed to create your server's DNS records" default="">
+#<UDF name="subdomain" label="Subdomain" example="The subdomain for the DNS record: www (Requires Domain)" default="">
+#<UDF name="domain" label="Domain" example="The domain for the DNS record: example.com (Requires API token)" default="">
+
+# git repo
+export GIT_REPO="https://github.com/akamai-compute-marketplace/marketplace-apps.git"
+export WORK_DIR="/tmp/marketplace-apps" 
+export MARKETPLACE_APP="apps/linode-marketplace-influxdb"
+
+# enable logging
+exec > >(tee /dev/ttyS0 /var/log/stackscript.log) 2>&1
+
+function cleanup {
+  if [ -d "${WORK_DIR}" ]; then
+    rm -rf ${WORK_DIR}
+  fi
+
+}
+
+function udf {
+  local group_vars="${WORK_DIR}/${MARKETPLACE_APP}/group_vars/linode/vars"
+  sed 's/  //g' <<EOF > ${group_vars}
+
+  # deployment vars
+  soa_email_address: ${SOA_EMAIL_ADDRESS}
```

Comment:
```suggestion
  soa_email_address: ${SOA_EMAIL_ADDRESS}
  admin_username: ${ADMIN_USERNAME}
```

The playbook fails because admin_username is undef.

Reply:
Corrected in [b62ab04](https://github.com/akamai-compute-marketplace/marketplace-apps/pull/120/commits/b62ab040728a1c112d0f5894c307bfd2f1dd3f3c).